site/content/en/topics/assumptions.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Assumptions for Provisioning Tools"
+linkTitle: "Assumptions for Provisioning Tools"
+weight: 1
+type: docs
+description: >
+    Cloud provider assumptions on Azure resources that provisioning tools should take care.
+---
+
+Here is a list of Azure resource assumptions that are required for cloud provider Azure:
+
+* All Azure resources are under the same tenant.
+* All the virtual machines are part of either VirtualMachineScaleSet or AvailabilitySet.
+* All virtual machine's name should be the same as their hostname.
+* LoadBalancer's name should be `<clusterName>` for external type and `<clusterName>-internal` for internal type.
```

Comment:
What about multiple SLB scenario? In the future, the expected way would be:
1. each VMSS or VMAS owns an external SLB;
2. only the SLB of the primary VMSS or VMAS would be named `<clusterName>`, others would be named `<vm_set_name>`.

Reply:
let me add a set for it.